### ISSUE TITLE
something is wrong with app.css

### ISSUE THREAD

**Issue by diomed** [robbinfellow/haywire-grav#14]
it doesn't load bulma @ all in latest version of theme.
@ least not for me.

**Comment by robbinfellow** [comment]
Hm, it's working on my end. Did you run `yarn && yarn production` from the theme folder? If you didn't change anything it should be working.

Could you provide steps in order to reproduce the issue please?

**Comment by diomed** [comment]
I just added theme via GPM

**Comment by robbinfellow** [comment]
Ill try installing with GPM, brb

**Comment by TiagoFonseca** [comment]
I followed all the steps and after running `yarn && yarn production` I can see that all the URLs to css and js files from this theme are incorrect. My installation of Grav is in a sub folder (grav/grav-admin) and that path is being added twice to the URL.
eg: 
`localhost/grav/grav-admin/grav/grav-admin/user/themes/haywire/dist/css/... `
instead of 
`localhost/grav/grav-admin/user/themes/haywire/dist/css/...`
Do you know what can be causing this?

**Comment by robbinfellow** [comment]
The asset path is set in [haywire.yaml](https://github.com/robbinfellow/haywire-grav/blob/4755f92dc9c21e7c5ccecda494cf1733907c1269/haywire.yaml#L1).

My guess is that this issue may be caused by the asset cache-busting I implemented recently, see [haywire.php](https://github.com/robbinfellow/haywire-grav/blob/develop/haywire.php).

Maybe this is a bad way of doing this type of thing, I'm not super-up-to-date when it comes to Grav. One question though, why are you installing in a sub-folder? I didn't know that was a thing, is that something new/something I missed?

When I'm testing this _(with no issues)_ I'm installing the theme in the following path `grav/user/themes/`. Feel free to correct me if there's something I miss here! 😕 

Thanks for using the theme!

**Comment by robbinfellow** [comment]
Ok, I double-checked this issue again, did a fresh install of grav _(via composer)_ + haywire theme _(both via GPM and via git clone)_ and I still cannot reproduce the issues both of you are describing with assets not being loaded as it should/having a faulty url.

Could you please provide some steps to reproduce this issue for me so that I can get something more to go on?

**Comment by diomed** [comment]
I forgot to mention something maybe important, that I'm running Grav v1.4.0-rc.1 ATM
This is the link that wont open @ all:
`<link href="/grav/grav/user/themes/haywire/dist/css/app.css?id=1f1500e21b66c2b2aa54" type="text/css" rel="stylesheet" />`

(installation is in grav directory)

any other installed theme works tho.

**Comment by TiagoFonseca** [comment]
I didn't need the sub-folder, it was a mistake. So I installed Grav again, so now it's in htdocs/grav. I also installed haywire theme and I have the same problem as @diomed, it's looking for the haywire css file under 
`<link href="/grav/grav/user/themes/haywire/dist/css/app.css?id=1f1500e21b66c2b2aa54" type="text/css" rel="stylesheet" />` 
and the js under 
`<script src="/grav/grav/user/themes/haywire/dist/js/app.js?id=ab0684f27c33a69246c2" type="text/javascript"></script>`
it seems to me that if I don't put grav in the root of my server that this won't work.

**Comment by robbinfellow** [comment]
@diomed @TiagoFonseca 

I managed to reproduce the issue following the steps you provided, I will look into this issue and hopefully have a fix ready for you soon!

Sorry for the inconvenience with the asset cache busting update. I will let you know when there's an update/fix for this issue.

**Comment by TiagoFonseca** [comment]
Thanks for your hardwork building this theme and for looking into this issue. I hope you can find a fix soon.

**Comment by robbinfellow** [comment]
I've added a **_possible fix_** (28fce37) for this issue, I'm not 100% sure it works in your environment, had some issues setting up a similar dev environment using **Laravel Valet**. 

Would you mind testing in your environment to see if the problem still exists? @TiagoFonseca @diomed 

You'd have to clone `develop` branch:
```shell
$ cd user/themes/
$ git clone -b develop https://git@github.com/robbinfellow/haywire-grav.git
$ yarn && yarn production
```

Please let me know if it works or not!

**Comment by robbinfellow** [comment]
#### Update:

Fixed in `0.3.4`. 

Go ahead and update:

`$ bin/gpm update haywire`
